server/routes/routes.js

```diff
@@ -48,20 +61,58 @@ const router = (app) => {
             reward,
             resources,
             by_time,
-            by_day,
+            by_date,
             sub_task_option,
             sub_tasks,
           ])
+          .then((result) => {
+            res.status(204);
+          })
           .catch((e) => console.error(e));
       })
       .catch((e) => console.error(e));
+  })
+  .catch((e) => console.error(e));
   });
 
-  // Catch all - non matching routes sent back to index.html (keep below all other routes and above err handling).
-  app.get("*", (req, res) => {
-    res.sendFile(path.join(__dirname, "../../client/build/index.html"));
+  // Edit user tasks 
+  app.put("/api/tasks", (req, res) => {
+    // console.log(req.session.passport.user);
+    console.log(req.body);
+    const subject_id = req.body.id;
+    const task_subject = req.body.task_subject;
+    const subject_description = req.body.subject_description;
+    const reward = req.body.reward;
+    const resources = req.body.resources;
+    const by_time = req.body.by_time;
+    const by_date = req.body.by_date;
+    const sub_task_option = req.body.sub_task_option;
+    const sub_tasks = req.body.sub_tasks;
+    const query =
```

Comment:
It may be possible here too, lines 83 to 90.

Reply:
same here